doc/vlan/switchport-mode-support/Switchport Mode and VLAN CLI Enhancement.md

```diff
@@ -0,0 +1,608 @@
+  
+ ```
+ ![PortChannel-access](https://user-images.githubusercontent.com/61490193/223190449-b0b7eab3-ba0b-4c82-baa3-988ae22a787d.png)
+
+
+
+ ```
+  admin@sonic:~$ sudo config switchport mode trunk PortChannel0002
+ 
+  Usage:  This command will change PortChannel0002 mode from routed to trunk
+  
+ ```
+ ![PortChannel-trunk](https://user-images.githubusercontent.com/61490193/223190536-16ad5af8-ebf8-481a-932a-8879cecba133.png)
+
+
+After configuring port and port channel to “access” and “trunk”. We have added untagged and tagged members on the ports and portchannel using the same commands as has been used before. 
+
+
+![Vlan-Brief](https://user-images.githubusercontent.com/61490193/223190727-b478448a-806f-4020-bd8f-71dc3eddedd7.png)
+
+
+####  Displaying switchport modes on Port & Port Channel
+
+Show interface status output is modified, the column header named “Vlan” is renamed as “Mode” to depict “access” and “trunk” modes.
+
+Existing Output:
+
+![Show-Int-Stat-old](https://user-images.githubusercontent.com/61490193/223190797-d47cf8fb-e6c6-406a-bb73-b0af8721e3b4.png)
+
+
+Modified Output:
+
+
+![New-Int-Stat](https://user-images.githubusercontent.com/61490193/223196559-e4666118-d022-4a23-85f0-f0ee96d61654.png)
+
+
+#### Switching Ports & PortChannels mode from Access to Routed/Trunk
+
+We are giving examples for physical ports, these commands work in the same way for PortChannels as they do for physical ports.
+
+#### Switch from access to routed
+In Switching from access to routed, user need to remove the untagged member first otherwise, the command will give an error:
+
+![Switch-Access-Route](https://user-images.githubusercontent.com/61490193/223190966-7498a860-98d0-456f-8acb-241612a51790.png)
+
+
+#### Switch from access to trunk
+Switching from access to trunk will work without any error and untagged member added in the access mode is retained:
+
+![Switch-Access-Trunk](https://user-images.githubusercontent.com/61490193/223190991-aeef8fd6-e71a-4bbf-9862-42ea4613002a.png)
+
+To verify the port mode user can make of “show interfaces status” command
+
+
+#### Switching Ports & PortChannels mode from Trunk to Routed/Access
+
+#### Switch from trunk to routed
+Switch from trunk to routed user need to remove the untagged/tagged member(s) first otherwise, the command will give an error:
+
+![Switch-Trunk-Routed](https://user-images.githubusercontent.com/61490193/223191099-c037754d-c221-43b6-874e-6dbcd4eae639.png)
+
+
+#### Switch from trunk to access
+Switch from trunk to access, user need to remove the tagged members firstotherwise, the command will give an error:
+
+![Switch-Trunk-Access](https://user-images.githubusercontent.com/61490193/223191151-cea02ab6-7755-4c7c-994c-72bbbaeccc56.png)
+
+
+
+###  Examples/Usage of VLAN CLI Enchancemnts Command
+
+ #### Add multiple Vlan(s) using a single command:
+
+```
+  admin@sonic:~$ sudo config vlan add -m 100,200,300
+
+  Usage:  This command will add multiple comma separated vlan in a list
```

Comment:
If you show the errors being shown for each error conditions would be helpful.

Reply:
Sure we will update it.